### PR TITLE
Using symfony/console

### PR DESCRIPTION
**Pull request by Chemaclass**
## 📚 Description

Using Symfony Command Component for our Commands as well. 

## 🔖 Changes

- Require `symfony/console`
- Extend Symfony Command in the Export, Format, Repl, Run and Test phel commands
- Instantiate the phel commands directly in the phel entry point
- Deleted the unnecessary PhelFacade/PhelFactory code. It's not necessary anymore.

### Follow up

I got some ideas for some other related clean-ups on the commands, but I will apply them in other PR. On this PR I wanted to do the few changes as possible because it would get too big (and out of control) easily. This way it will be easier to see what's going on.

### CODE REVIEW

**Review by JesusValera — APPROVED** [review]
When I saw the PR title I was not sure about the idea of including the Symfony dependency in the project.
But now with perspective I find much explicit and understandable the command. 👍🏼👍🏼😀

File: src/php/Command/Export/ExportCommand.php [hunk 1]
```diff
@@ -51,6 +60,8 @@ public function run(): void
         } catch (Throwable $e) {
             $this->io->writeStackTrace($e);
```
Comment (jenshaase) [inline src/php/Command/Export/ExportCommand.php:61]: `return Command::FAILURE;`

File: src/php/Command/Run/RunCommand.php [hunk 2]
```diff
@@ -23,26 +27,41 @@ public function __construct(
...
 
-    public function run(string $fileOrPath): void
+    public function execute(InputInterface $input, OutputInterface $output): int
     {
+        /** @var string $fileOrPath */
+        $fileOrPath = $input->getArgument('path');
         try {
             $this->loadNamespace($fileOrPath);
         } catch (CompilerException $e) {
             $this->io->writeLocatedException($e->getNestedException(), $e->getCodeSnippet());
```
Comment (jenshaase) [inline src/php/Command/Run/RunCommand.php:59]: `return Command::FAILURE;`

File: src/php/Command/Run/RunCommand.php [hunk 3]
```diff
@@ -23,26 +27,41 @@ public function __construct(
...
         } catch (Throwable $e) {
             $this->io->writeStackTrace($e);
```
Comment (jenshaase) [inline src/php/Command/Run/RunCommand.php:61]: `return Command::FAILURE;`

File: src/php/Command/Run/RunCommand.php [hunk 4]
```diff
@@ -23,26 +27,41 @@ public function __construct(
...
         }
+
+        return 0;
```
Comment (jenshaase) [inline src/php/Command/Run/RunCommand.php:64]: `return Command::SUCCESS;`

File: src/php/Command/Repl/ReplCommand.php [hunk 5]
```diff
@@ -39,27 +42,35 @@ public function __construct(
...
-    public function run(): void
+    public function execute(InputInterface $input, OutputInterface $output): int
     {
         $this->io->readHistory();
         $this->io->writeln($this->style->yellow('Welcome to the Phel Repl'));
         $this->io->writeln('Type "exit" or press Ctrl-D to exit.');
 
         $this->loopReadLineAndAnalyze();
+
+        return 0;
```
Comment (jenshaase) [inline src/php/Command/Repl/ReplCommand.php:73]: `return Command::SUCCESS;`

File: src/php/Command/Format/FormatCommand.php [hunk 6]
```diff
@@ -51,6 +65,8 @@ public function run(array $paths): void
         }
 
         $this->printResult();
+
+        return 0;
```
Comment (jenshaase) [inline src/php/Command/Format/FormatCommand.php:69]: `return Command::SUCCESS;`

File: src/php/Command/Export/ExportCommand.php [hunk 7]
```diff
@@ -51,6 +60,8 @@ public function run(): void
...
         }
+
+        return 0;
```
Comment (jenshaase) [inline src/php/Command/Export/ExportCommand.php:64]: `return Command::SUCCESS;`

**Review by jenshaase — CHANGES_REQUESTED** [review]

**Review by jenshaase — APPROVED** [review]

**Comment by jenshaase** [comment]
Thanks, I merged it!

**Pull request merged**